hpo_demo_setup.sh

```diff
@@ -0,0 +1,288 @@
+#!/bin/bash
+#
+# Copyright (c) 2020, 2022 Red Hat, IBM Corporation and others.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Default docker image repos
+HPO_DOCKER_REPO="docker.io/kruize/hpo"
+HPO_VERSION="0.01"
```

Comment:
We should not be hard coding versions here. Instead it should pick up the latest HPO version from some artifact in the HPO repo itself

Reply:
Also this should be HPO version `0.0.1` and not `0.01`